common/shared.go

```diff
@@ -4,10 +4,10 @@ const (
 	ConfigPerms   = 0750
 	SlotsPerEpoch = 32
 
-	GethTag        = "1.13.12"
-	ErigonTag      = "2.57.3"
-	PrysmTag       = "v4.2.1"
-	LighthouseTag  = "v4.6.0"
-	TekuTag        = "24.1.1"
-	GethCommitHash = "02eb36af"
+	GethTag        = "1.13.14"
+	ErigonTag      = "2.58.2"
+	PrysmTag       = "v5.0.1"
+	LighthouseTag  = "v5.1.1"
+	TekuTag        = "24.3.0"
+	GethCommitHash = "2bd6bd0"
```

Comment:
For some reason github provides only first 7 characters of the release commit hash - this needs to contain 8 of those, so this would be `2bd6bd01`

Reply:
Ok, thanks!

Reply:
Changed.